.github/workflows/ci.yml

```diff
@@ -126,7 +126,7 @@ jobs:
     - name: Run the tests with x509-parser enabled
       run: cargo test --verbose --features x509-parser --all-targets
     - name: Run the tests with no default features enabled
-      run: cargo test --verbose --no-default-features --all-targets
+      run: cargo test --verbose --all-targets
```

Comment:
It seems to me that since the choice of crypto provider is done via features, and *ring* overrides aws-lc-rs, that we need CI tasks for `--no-default-features --features=aws_lc_rs` to prevent bitrot there. The `--all-features` and default features builds will both be testing *ring*.